script/Ciel/Day 3/03_01_CIEL03_3_3B.txt

```diff
@@ -419,7 +419,7 @@ A different kind of heartache.
 -- Page 219, Offset 5416.
 -- 先輩の気遣いに心が折れそうで、うつむいて息を吐く。
 My heart feels like it's about to break under the weight of Senpai's concern,
- so I look away and exhale deeply.
+ so I look away, and exhale deeply.
```

Comment:
On second glance, I don't think this needs a comma. It'd have to be changed to something like "away, exhaling deeply" to justify it.